website/pages/docs/rules.mdx

```diff
@@ -10,7 +10,7 @@ import { AutomaticModeWarning } from '../../components/automatic-mode-warning';
 
 <AutomaticModeWarning />
 
-You are probably here because you got a warning message in your console.
+When working with blocks, there are 3 common reasons you may encounter errors:
 
 <Callout type="warning">
```

Comment:
Not sure this needs to be an aside / what the purpose of this is.

If you want to show what an error message looks like, might be better to show what the console errors may look like.